conf/gnss-sdr_GPS_L1_monitor.conf

```diff
@@ -0,0 +1,90 @@
+; This is a GNSS-SDR configuration file
+; The configuration API is described at https://gnss-sdr.org/docs/sp-blocks/
+; SPDX-License-Identifier: GPL-3.0-or-later
+; SPDX-FileCopyrightText: (C) 2010-2020  (see AUTHORS file for a list of contributors)
+
+[GNSS-SDR]
+
+;######### GLOBAL OPTIONS ##################
+GNSS-SDR.internal_fs_sps=2000000
+
+;######### SIGNAL_SOURCE CONFIG ############
+SignalSource.implementation=File_Signal_Source
+SignalSource.filename=/tmp/2013_04_04_GNSS_SIGNAL_at_CTTC_SPAIN.dat
+SignalSource.item_type=ishort
+SignalSource.sampling_frequency=4000000
+SignalSource.freq=1575420000
+SignalSource.samples=0
+SignalSource.enable_throttle_control=true
+
+;######### SIGNAL_CONDITIONER CONFIG ############
+SignalConditioner.implementation=Signal_Conditioner
+
+;######### DATA_TYPE_ADAPTER CONFIG ############
+DataTypeAdapter.implementation=Ishort_To_Complex
+
+;######### INPUT_FILTER CONFIG ############
+InputFilter.implementation=Pass_Through
+InputFilter.item_type=gr_complex
+
+;######### RESAMPLER CONFIG ############
+Resampler.implementation=Direct_Resampler
+Resampler.sample_freq_in=4000000
+Resampler.sample_freq_out=2000000
+Resampler.item_type=gr_complex
+
+;######### CHANNELS GLOBAL CONFIG ############
+Channels_1C.count=3
+Channels.in_acquisition=1
+Channel.signal=1C
+Channel0.satellite=1
+Channel1.satellite=11
+Channel2.satellite=17
+
+;######### ACQUISITION GLOBAL CONFIG ############
+Acquisition_1C.implementation=GPS_L1_CA_PCPS_Acquisition
+Acquisition_1C.item_type=gr_complex
+Acquisition_1C.threshold=0.008
+Acquisition_1C.doppler_max=10000
+Acquisition_1C.doppler_step=250
+
+;######### TRACKING GLOBAL CONFIG ############
+Tracking_1C.implementation=GPS_L1_CA_DLL_PLL_Tracking
+Tracking_1C.item_type=gr_complex
+Tracking_1C.pll_bw_hz=40.0;
+Tracking_1C.dll_bw_hz=4.0;
+
+;######### TELEMETRY DECODER GPS CONFIG ############
+TelemetryDecoder_1C.implementation=GPS_L1_CA_Telemetry_Decoder
+
+;######### OBSERVABLES CONFIG ############
+Observables.implementation=Hybrid_Observables
+
+;######### PVT CONFIG ############
+PVT.implementation=RTKLIB_PVT
+PVT.averaging_depth=100
+PVT.flag_averaging=true
+PVT.output_rate_ms=10
+PVT.display_rate_ms=500
+PVT.enable_monitor=true
+PVT.decimation_factor=1
+PVT.client_addresses=127.0.0.1
+PVT.udp_port=1234
```

Comment:
The highlighted configuration options for the [PVT monitoring port](https://gnss-sdr.org/docs/sp-blocks/pvt/#custom-streaming) should be prepended with `monitor_` after the `PVT.` part:

```
PVT.monitor_decimation_factor=1
PVT.monitor_client_addresses=127.0.0.1
PVT.monitor_udp_port=1234
```

Reply:
Actually, there is no `PVT.monitor_decimation_factor` option, so it should be removed.

Reply:
Fixed with commit 37a9e24!